packages/valko-ui-components/src/types/Input.ts

```diff
@@ -1,9 +1,9 @@
-import type { Sizes, ColorsWithSurface, Variants, Shapes } from './common'
+import type { Sizes, ColorsWithSurface, Variants, Shapes, AriaAttributes } from './common'
 
 export type InputType = 'text' | 'password' | 'email' | 'number'
 export type InputCursor = 'text' | 'pointer'
 
-export interface InputProps extends Sizes, ColorsWithSurface, Variants, Shapes {
+export interface InputProps extends Sizes, ColorsWithSurface, Variants, Shapes, Pick<AriaAttributes, 'aria-label' | 'aria-labelledby' | 'aria-describedby' | 'aria-invalid' | 'aria-required'> {
```

Comment:
Check if this is shorter using Omit instead of Pick, since you are picking a lot of props